scripts/ethEndToEndTest.js

```diff
@@ -100,7 +100,8 @@ async function main() {
                 for (const {event: {method, section, data}} of events) {
                     console.log('\t', `: ${section}.${method}`, data.toString());
                     const [, claimer] = data;
-                    if (section === 'erc20Peg' && method == 'Erc20Claim' && claimer && claimer.toString() === alice.address) {
+                    if (section === 'erc20Peg' && method == 'Erc20Claim' && claimer &&
+                        (claimer.toString() === alice.address || claimer.toString() === encodeAddress(claim.beneficiary))) {
```

Comment:
what is this additional condition here for @KarishmaBothara ? `claimer.toString() === encodeAddress(claim.beneficiary)`